### PR TITLE
Benchmark - Support batch/shape range in cublaslt gemm

### PR DESCRIPTION
**Pull request by abuccts**
Support batch and shape range with multiplication factors in cublaslt gemm benchmark.

### CODE REVIEW

File: superbench/benchmarks/micro_benchmarks/cublaslt_function.py [hunk 1]
```diff
@@ -57,9 +92,10 @@ def add_parser_arguments(self):
...
         )
         self._parser.add_argument(
-            '--in_type',
+            '--in_types',
             type=str,
-            default='fp8e4m3',
+            nargs='+',
+            default=['fp8e4m3'],
             required=False,
             help='Input data type, supports {}.'.format(' '.join(self._in_types)),
```
Comment (cp5555) [inline superbench/benchmarks/micro_benchmarks/cublaslt_function.py:100]: Shall we revise the help to explain multiple support.

**Review by cp5555 — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/cublaslt_function.py (hunk 1 above)
Comment (abuccts) [reply superbench/benchmarks/micro_benchmarks/cublaslt_function.py]: yes, updated

File: superbench/benchmarks/micro_benchmarks/cublaslt_function.py [hunk 2]
```diff
@@ -75,20 +111,28 @@ def _preprocess(self):
...
-                f'{self.__bin_path} -m {shape_list[0]} -n {shape_list[1]} -k {shape_list[2]} '
-                f'-b {self._args.batch} -w {self._args.num_warmup} -i {self._args.num_steps} -t {self._args.in_type}'
-            )
+            for _b in self.mrange(*map(int, self._args.batch.split(':'))):
+                for shape in self._args.shapes:
+                    shape_list = shape.replace(',', ' ').split()
+                    if len(shape_list) != 3 or not all(self.validate_mrange(x) for x in shape_list):
+                        logger.error(f'Invalid shape {shape}.')
+                        return False
+                    for _m, _n, _k in itertools.product(
```
Comment (yukirora) [inline superbench/benchmarks/micro_benchmarks/cublaslt_function.py:129]: what if m or n or k is 0?

Comment (abuccts) [reply superbench/benchmarks/micro_benchmarks/cublaslt_function.py:129]: 0 or 1 or other unsupported shapes will get "cuBLAS API failed" in runtime

**Pull request merged**